utils/ccloud_library.sh

```diff
@@ -769,11 +769,29 @@ function ccloud::create_acls_connect_topics() {
   serviceAccount=$1
 
   echo "Connect: creating topics and ACLs for service account $serviceAccount"
-  for topic in connect-demo-configs connect-demo-offsets connect-demo-statuses _confluent-monitoring _confluent-command ; do
-    ccloud kafka topic create $topic &>/dev/null
-    ccloud kafka acl create --allow --service-account $serviceAccount --operation WRITE --topic $topic --prefix
-    ccloud kafka acl create --allow --service-account $serviceAccount --operation READ --topic $topic --prefix
+
+  TOPIC=connect-demo-configs
+  ccloud kafka topic create $TOPIC --partitions 1 --config "cleanup.policy=compact"
```

Comment:
@rspurgeon I understand and agree with the objective here is to configure different partition counts for each of the connect topics.  

1. Can you please elaborate on how partitions were selected and known to be good as 1/6/3 for each respective one?
2. How did we miss this?  Do you think I forgot to validate this version of the demo (perhaps only ran the Docker version) or did some behavior change?

Reply:
1. I followed this guide: https://docs.confluent.io/current/connect/references/allconfigs.html#common-worker-configuration  I did not follow it exactly, however, as CCloud charges per partition, so adding a lot of partitions for topics seems like bad form for exploratory users.  I used instinct for the 6 and 3 respectively, the 1 is required on the configs.
2. I'm not clear.  I'm not sure what other demos exist that run local connect workers to cloud and could possibly have been affected.  Assuming other ones are using the ccloud stack functions, they should be covered by this.

Reply:
@rspurgeon ack and thanks